packages/ui/src/views/repo/pull-request/details/components/conversation/pull-request-comment-box.tsx

```diff
@@ -164,6 +175,7 @@ export const PullRequestCommentBox = ({
   useEffect(() => {
     if (textAreaRef.current) {
       textAreaRef.current.setSelectionRange(textSelection.start, textSelection.end)
+      textAreaRef.current.focus()
```

Comment:
@jake-bassett This will bring back autofocus to the comment box when the page loads, the comment box will be focused by default, causing the page to scroll to that section.

Can we update the logic to focus only when it's not the initial mount of the component?

Reply:
Ah, good call out. Updated to only set focus after action click.